build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/plugin_list.rb

```diff
@@ -63,7 +63,8 @@ def default_plugins
         'Aws::Plugins::ChecksumAlgorithm' => "#{core_plugins}/checksum_algorithm.rb",
         'Aws::Plugins::RequestCompression' => "#{core_plugins}/request_compression.rb",
         'Aws::Plugins::DefaultsMode' => "#{core_plugins}/defaults_mode.rb",
-        'Aws::Plugins::RecursionDetection' => "#{core_plugins}/recursion_detection.rb"
+        'Aws::Plugins::RecursionDetection' => "#{core_plugins}/recursion_detection.rb",
+        'Aws::Plugins::Telemetry' => "#{core_plugins}/telemetry.rb"
```

Comment:
I think we'll need to update the min core version for all gems (otherwise this may fail for users with old versions of core).